test/e2e-common.sh

```diff
@@ -193,13 +165,12 @@ function install_operator() {
 # Uninstalls Knative Serving from the current cluster.
 function knative_teardown() {
   echo ">> Uninstalling Knative serving"
-  echo "Istio YAML: ${INSTALL_ISTIO_YAML}"
   echo ">> Bringing down Serving"
   kubectl delete -n $TEST_NAMESPACE KnativeServing --all
   echo ">> Bringing down Eventing"
   kubectl delete -n $TEST_NAMESPACE KnativeEventing --all
   echo ">> Bringing down Istio"
-  kubectl delete --ignore-not-found=true -f "${INSTALL_ISTIO_YAML}" || return 1
+  $HOME/istioctl x uninstall --purge
```

Comment:
$HOME/.istioctl/bin/istioctl or $HOME/istioctl????

Reply:
Thanks! It was mistake. Fixed.